tests/test_31_oauth2_introspection.py

```diff
@@ -225,7 +224,7 @@ def _mint_token(self, token_class, grant, session_id, based_on=None, **kwargs):
             endpoint_context=self.token_endpoint.server_get("endpoint_context"),
             token_class=token_class,
             token_handler=self.session_manager.token_handler.handler[token_class],
-            expires_at=time_sans_frac() + 300,  # 5 minutes from now
+            expires_at=utc_time_sans_frac() + 300,  # 5 minutes from now
```

Comment:
@rohe this 300 must be defined in the oidcop configuration together with a timeslack for verifications

Reply:
OK, I'll do it.

Reply:
The proper name is maximum clock skew :-)